### ISSUE TITLE
DaddyLive only showing 184 channels

### ISSUE THREAD

**Issue by joelstitch** [cabernetwork/cabernet#142]
I just install cabernet and enabled the daddyLive plugin and the TV Guide plugin. I updated the channel list and EPG a few  times but DaddyLive only shows 184 channels. It just shows me channels from A-D. Is there anything I need to do to make it show the rest of the channels?

**Comment by rocky4546** [comment]
Where do you see the 184 channels?  I assume it's in the channel editor with that number of channels.  If not, please provide the number of channels listed in the channel editor.

**Comment by joelstitch** [comment]
Yes, the channel editor. I also have it connected to Plex and it shows the same amount on plex.
